commands/virtual_simulation/battle.js

```diff
@@ -130,6 +130,7 @@ export default async (client, interaction) => {
                         text += addLine(`**${nicks[h]}** won ${exp[0]} exp. points!`);
                         if (exp[1] > 0) {
                             text += addLine(`**${nicks[h]}** grew to level **${shinxes[h].level}**!`);
+                            shinxApi.addBattleRewards(trainers[h].id, shinxes[h].level);
```

Comment:
Should battle rewards not be checked on every exp gain instead of every battle?
Unless battling is the only way to get exp forever.